docs/plugins/index.rst

```diff
@@ -540,3 +543,4 @@ Here are a few of the plugins written by the beets community:
 .. _beets-originquery: https://github.com/x1ppy/beets-originquery
 .. _drop2beets: https://github.com/martinkirch/drop2beets
 .. _beets-audible: https://github.com/Neurrone/beets-audible
+.. _beets-more: https://forgejo.sny.sh/sun/beetsplug
```

Comment:
Looks like you have all the plugins in the same folder. Can you please separate it out into a separate plugin so that users can install only the plugin they want? It is just a matter of reorganizing the files.

Reply:
Sure. I also updated the URL to point to the specific plugin directory.